File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/QuadrosSocietariosVM.cs
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
    public class QuadrosSocietariosVM
    {
        public int? IdPessoaJuridica { get; set; }
        public IEnumerable<QuadroSocietarioVM> QuadrosSocietarios { get; set; }
    }
}

File: src/Suframa.Sciex.UserInterface/www/app/view-model/QuadrosSocietariosVM.ts
import { quadroSocietarioVM } from './QuadroSocietarioVM';

export class quadrosSocietariosVM {
	constructor() {
		this.quadrosSocietarios = new Array<quadroSocietarioVM>();
	}

	idPessoaJuridica?: number;
	quadrosSocietarios: quadroSocietarioVM[];
}
